plutus-core/plutus-ir/test/transform/inline/letAppMulti

```diff
@@ -0,0 +1,40 @@
+{- 
+Multiple occurrences of a let binding that is a function application (funApp). It should not get 
+inlined by the unconditional inlining optimization. However, the optimization that inlines fully
+applied function should inline funApp.
```

Comment:
I don't think so. If you inline `[ funApp appNum ]`, it becomes `[ [ (builtin addInteger) appNum ] appNum ]`, which is bigger, so it makes no sense.

Reply:
Looking at the [code](https://github.com/input-output-hk/plutus/blob/ed44f958a37f126b2aaa33cd3569691d1c32bbb7/plutus-core/plutus-ir/src/PlutusIR/Transform/Inline/Utils.hs#L211) it seems builtins are cost and size acceptable? Also, I think it makes sense to accept builtins, especially seeing [this](https://gist.github.com/thealmarty/a4facd1ed3b25302eb60497c1f10a696).

Reply:
Although applications are not cost and size acceptable, so it should not get inlined, as desired.